kubernetes-deploy.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
   spec.add_dependency "activesupport", ">= 4.2"
+  spec.add_dependency "kubeclient", "~> 2.3"
```

Comment:
I suspect this version is different from what we used before. When I clone down this branch to attempt to run the tests I get:

```
macbookbro kubernetes-deploy (restart-task) $ rt test/integration/restart_task_test.rb 
Run options: --seed 51416

# Running:

/Users/sirup/src/github.com/Shopify/kubernetes-deploy/test/test_helper.rb:65:in `create_namespace': uninitialized constant Kubeclient::Namespace (NameError)
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/test/test_helper.rb:57:in `claim_namespace'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/test/test_helper.rb:44:in `run'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:822:in `run_one_method'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:323:in `run_one_method'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:310:in `block (2 levels) in run'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:309:in `each'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:309:in `block in run'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:349:in `on_signal'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:336:in `with_info_handler'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:308:in `run'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:158:in `block in __run'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:158:in `map'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:158:in `__run'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:135:in `run'
	from /Users/sirup/src/github.com/Shopify/kubernetes-deploy/vendor/bundle/gems/minitest-5.10.1/lib/minitest.rb:62:in `block in autorun'
```